src/data/quality-assurance-quiz.ts

```diff
@@ -1,4 +1,14 @@
 const qualityAssuranceQuiz = [
+  {
+    Question: "What are web accessibility evaluation tools?",
+    Answer: "Software programs or online services that help determine if web content meets accessibility standards.",
+    Distractor1: "Tools that help determine if unit tests are well written.",
+    Distractor2: "Software programs that help determine if a website is secure.",
+    Distractor3: "Online services that help determine if customer satisfaction is high.",
+    Explanation:
+      "Web accessibility evaluation tools are software programs or online services that help determine if web content meets accessibility standards.",
+    Link: "https://www.w3.org/WAI/test-evaluate/tools/",
+  },
```

Comment:
This question looks good to me.
Quincy approved the new accessibility category.
I just pushed up a change to add that new category.

Please move this question over to the accessibility quiz. 👍